extensions/datastores/kudu/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>geowave-extension-parent</artifactId>
+		<groupId>org.locationtech.geowave</groupId>
+		<relativePath>../../</relativePath>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>geowave-datastore-kudu</artifactId>
+	<name>GeoWave Kudu</name>
+	<description>Geowave Data Store on Apache Kudu</description>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.locationtech.geowave</groupId>
+			<artifactId>geowave-core-mapreduce</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.locationtech.geowave</groupId>
+			<artifactId>geowave-core-cli</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.kudu</groupId>
+			<artifactId>kudu-client</artifactId>
+			<version>1.8.0</version>
```

Comment:
It seems that this convention is not followed in a few of the other datastore extensions, in particular, with the client. Should I modify those as well?
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/dynamodb/pom.xml#L30
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/cassandra/pom.xml#L27
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/redis/pom.xml#L44

with some that don't have a pinned version:
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/hbase/pom.xml#L49
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/rocksdb/pom.xml#L28
https://github.com/locationtech/geowave/blob/ecea4c417785a49947da4f6b239a39c3fa51bbe3/extensions/datastores/accumulo/pom.xml#L47

Reply:
the first three, yeah, the next 3 look fine to me (the version is in the "dependencyManagement" tag of the root pom which is actually a nice place to put it so that it overrides any version conflicts that could come from transitives)